images/replication-job/README.md

```diff
@@ -1,6 +1,6 @@
 ### Delta replications job container
 
-This contain is responsible for creating the delta replication files it may be set up by minute, hour, etc. Those replications files will be uploaded to a repository in AWs of Gooble Storage, depends on what you are using.
+This contain is responsible for creating the delta replication files it may be set up by minute, hour, etc. Those replications files will be uploaded to a repository in AWS of Google Storage, depends on what you are using.
```

Comment:
Noticed another typo while reviewing this :D - that should be `uploaded to a repository in AWS **or** Google Storage` . cc @Rub21 can do as a separate PR.

Reply:
@batpad tnx for finding.